src/main/java/com/epam/lab/payments/services/PaymentsService.java

```diff
@@ -76,6 +80,13 @@
         return accountMapper.accountsToAccountsDto(bankAccountRepository.findAll());
     }
 
+    public Optional<BankAccountDTO> findOneBankAccount(Integer accountId) {
+        log.info("Account by id " + accountId + " was successfully found");
```

Comment:
I already told u, that u should use slf4j, and shouldn't use string concatenation.